docs/dev-tools/chopsticks/index.md

```diff
@@ -0,0 +1,437 @@
+
+When running a fork, it's accessible by default at:
+
+```bash
+ws://localhost:8000
+```
+
+You can interact with the forked chain using various [libraries](https://wiki.polkadot.network/docs/build-tools-index#libraries){target=_blank} such as [Polkadot.js](https://polkadot.js.org/docs/){target=_blank} and its user interface, [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=_blank}.
+
+### Using Polkadot.js Apps
+
+To interact with Chopsticks via the hosted user interface, visit [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=_blank} and follow these steps:
+
+1. Click the network icon in the top left corner
+2. Scroll to the bottom and select **Development**
+3. Choose **Custom**
+4. Enter `ws://localhost:8000` in the input field
+4. Click the **Switch** button
+
+![](/images/dev-tools/chopsticks/chopsticks-1.webp)
+![](/images/dev-tools/chopsticks/chopsticks-2.webp)
+
+You should now be connected to your local fork and can interact with it as you would with a real chain.
+
+### Using Polkadot.js Library
+
+For programmatic interaction, you can use the Polkadot.js library. Here's a basic example:
+
+```js
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+async function connectToFork() {
+  const wsProvider = new WsProvider('ws://localhost:8000');
+  const api = await ApiPromise.create({ provider: wsProvider });
+  
+  // Now you can use 'api' to interact with your fork
+  console.log(`Connected to chain: ${await api.rpc.system.chain()}`);
+}
+
+connectToFork();
+```
+
+## Replaying Blocks
+
+Chopsticks allows you to replay specific blocks from a chain, which is useful for debugging and analyzing state changes.  You can use the parameters in the [Configuration](#configuration) section to set up the chain configuration, and then use the run-block subcommand with additional options:
+
+|    Option     |           Description           |
+| :-----------: | :-----------------------------: |
+| `output-path` |    File path to print output    |
+|    `html`     | Generate html with storage diff |
+|    `open`     |       Open generated html       |
+
+For example, to replay block 1000 from Polkadot and save the output to a JSON file:
+
+```bash
+npx @acala-network/chopsticks run-block  \
+--endpoint wss://polkadot-rpc.dwellir.com  \
+--output-path ./polkadot-output.json  \
+--block 1000
+```
+
+## XCM Testing
+To test XCM (Cross-Consensus Messaging) messages between networks, you can fork multiple parachains and a relay chain locally using Chopsticks. 
+
+|    Option    |      Description       |
+| :----------: | :--------------------: |
+| `relaychain` | Relaychain config file |
+| `parachain`  | Parachain config file  |
+
+
+For example, to fork Moonbeam, Astar, and Polkadot enabling XCM between them, you can use the following command:
+
+```bash
+npx @acala-network/chopsticks xcm \
+--r polkadot \
+--p moonbeam \
+--p astar
+```
+
+After running it, you should see output similar to the following:
```

Comment:
@eshaben I can't decide whether it's better to leave it as a code block or use the terminal output. What do you think?

Reply:
@0xLucca I like the terminal better and for consistency across the docs